plugins/modules/ome_application_certificate.py

```diff
@@ -187,6 +188,13 @@ def get_resource_parameters(module):
     return method, uri, payload
 
 
+def get_san(subject_alternative_names):
+    if not subject_alternative_names:
+        return subject_alternative_names
+
+    return re.sub(r'\s*,\s*', ',', subject_alternative_names.strip())
```

Comment:
Can we use _replace_ instead of regular expression? Do you see any problem using replace function.

subject_alternative_names.replace(" ", "")

Reply:
yes we can, updated it